src/sentry/apidocs/parameters.py

```diff
@@ -256,13 +268,76 @@ class ProjectParams:
         description="Defaults to true where the behavior is to alert the user on every new issue. Setting this to false will turn this off and the user must create their own alerts to be notified of new issues.",
     )
 
+    DYNAMIC_SDK_LOADER_OPTIONS = OpenApiParameter(
+        name="dynamicSdkLoaderOptions",
+        location="query",
+        required=False,
+        type=inline_serializer(
+            name="DynamicSDKLoaderOptionsSerializer",
+            fields={
+                "hasReplay": serializers.BooleanField(required=False),
+                "hasPerformance": serializers.BooleanField(required=False),
+                "hasDebug": serializers.BooleanField(required=False),
+            },
+        ),
+        description="""
+Configures multiple options for the Javascript Loader Script.
+- `Performance Monitoring`
+- `Debug Bundles & Logging`
+- `Session Replay`: Note that the loader will load the ES6 bundle instead of the ES5 bundle.
+```json
+{
+    "dynamicSdkLoaderOptions": {
+        "hasReplay": true,
+        "hasPerformance": true,
+        "hasDebug": true
+    }
+}
+```
+""",
```

Comment:
nit: this lack of indenting looks weird? same with the string block lower in this file

Reply:
Tested it with indenting and it has to be this way or else it looks strange on the docs page

Reply:
i guess the linter didn't fix it so you're all good